src/packages.builds

```diff
@@ -24,9 +24,6 @@
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
     <!-- add specific builds / pkgproj's here to include in servicing builds -->
-    <Project Include="$(MSBuildThisFileDirectory)..\pkg\Microsoft.NETCore.Platforms\Microsoft.NETCore.Platforms.builds">
-      <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
-    </Project>
```

Comment:
we should probably keep this as we missed the platforms change

Reply:
can you verify that we update the stable version list in packageindex ?

Reply:
ok, I will close this PR as there is no change required for 2.1.28